core/templates/components/summary-tile/learner-topic-summary-tile.component.html

```diff
@@ -94,6 +94,7 @@
     box-shadow: 0 2px 2px 0 #00000040;
     height: 220px;
     padding: 0;
+    transition: all 300ms ease-out;
```

Comment:
Here and elsewhere, why are we adding a transition of 300ms?

Reply:
I just followed the [redesign](https://www.figma.com/design/YWe7SqfUVjxlJLKTUn0UZa/Project-2?node-id=4440-64869&t=B6apF8XilWBKjxJe-4). I can remove it if we want.

Reply:
I might be looking in the wrong place but I don't see any mention of transitions, can you please attach a screenshot? 

We should most likely remove the delay judging from the video, unless it was explicitly mentioned in the design file.

Reply:
<img width="1154" alt="Screenshot 2024-08-11 at 1 11 14 PM" src="https://github.com/user-attachments/assets/b7145b98-fe9e-4eee-a483-7b1d48f65d64">

Reply:
Ahh okay, I think I misunderstood the colours.

Is this when the button is focused on?
<img width="421" alt="image" src="https://github.com/user-attachments/assets/90ba2688-8e94-411b-92c8-0d3682c072fd">

Reply:
Yes. It's labeled as selected in the Figma. I used the focus pseudo-class; would it be better to use active instead?

Reply:
Thanks for the clarification! And nope, the focus pseudo-class is suitable in this case.